spec/load_spec.rb

```diff
@@ -7,17 +7,19 @@
 
   it "provides require_all functionality but using 'load' instead of 'require'" do
     require_all File.dirname(__FILE__) + '/fixtures/resolvable'
-    C.new.should be_cool
+    expect(C.new).to be_cool
 
     class C
+      remove_method :cool?
```

Comment:
Why is this needed? Wouldn't it change semantics of the original spec especially since `:cool?` is removed few lines later as well?